waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/SSIKit2WalletService.kt

```diff
@@ -550,16 +580,28 @@ class SSIKit2WalletService(
                 kmsType = EventDataNotAvailable
             )
         )
-        } else {
-            logger.warn { "Key delete operation not performed for alias: $alias" }
-            throw WebException(HttpStatusCode.BadRequest, "Failed to delete key: $alias")
-        }
-    }.onFailure {
-        val errorMessage = "Failed to delete key: ${it.message}"
-        logger.error(it) { errorMessage }
-        throw WebException(HttpStatusCode.BadRequest, errorMessage)
+
+        return signature
     }
 
+    override suspend fun verify(alias: String, signature: String): Boolean {
+        val key = JWKKey.importJWK(alias).getOrNull() ?: throw IllegalArgumentException("Key import failed")
+
+        val signature = key.verifyJws(signature)
```

Comment:
this has the same name as the parameter:
![image](https://github.com/user-attachments/assets/6c279d54-75be-4a48-bb8f-950d29e90e96)

not a problem here, since the parameter is not referenced below, but still can be confusing